www/js/controllers.js

```diff
@@ -115,8 +115,52 @@ angular.module('starter.controllers', [])
 .controller('FavouriteCtrl', function($scope, $stateParams) {
 
 })
+  
+.controller('ExploreCtrl', function($scope, $ionicFilterBar, geojsonService) {
+  $scope.dataset              = geojsonService.getData(); //get geojson data
+  $scope.datas                = $scope.dataset;           //duplicate set of data that is filtered by app
+  $scope.segmentSelectedIndex = 3;                        //stores current segment selection
```

Comment:
$scope.segmentSelectedIndex is not used in index.html. It can be a controller level variable, `var segmentSelectedIndex = 3;`.